This fixes the following compile problem with kernel 4.20:

In file included from arp.c:20:0:
include/linux/if_arp.h:121:16: error: 'IFNAMSIZ' undeclared here (not in a function)
  char  arp_dev[IFNAMSIZ];
                ^~~~~~~~
make[7]: *** [Makefile:459: arp.o] Error 1

This is caused by commit 6a12709da354 ("net: if_arp: use define instead 
of hard-coded value") in the upstream Linux kernel which is integrated 
in Linux 4.20.

--- a/src/oamd/io.c
+++ b/src/oamd/io.c
@@ -20,7 +20,6 @@
 #include <net/if.h>
 #include <netinet/in.h>
 #include <atm.h>
-#define _LINUX_NETDEVICE_H	/* glibc2 */
 #include <linux/types.h>
 #include <linux/if_arp.h>
 
--- a/src/arpd/itf.c
+++ b/src/arpd/itf.c
@@ -12,7 +12,6 @@
 #include <sys/types.h>
 #include <linux/atmclip.h>
 #include <sys/socket.h>
-#define _LINUX_NETDEVICE_H /* glibc2 */
 #include <linux/types.h>
 #include <linux/if_arp.h>
 
--- a/src/arpd/io.c
+++ b/src/arpd/io.c
@@ -21,7 +21,6 @@
 #include <atm.h>
 #include <linux/atmclip.h> /* for CLIP_DEFAULT_IDLETIMER */
 #include <linux/atmarp.h>
-#define _LINUX_NETDEVICE_H /* glibc2 */
 #include <linux/types.h>
 #include <linux/if_arp.h>
 
--- a/src/arpd/arp.c
+++ b/src/arpd/arp.c
@@ -15,7 +15,6 @@
 #include <sys/types.h>
 #include <sys/socket.h> /* for linux/if_arp.h */
 #include <netinet/in.h> /* for ntohs, etc. */
-#define _LINUX_NETDEVICE_H /* very crude hack for glibc2 */
 #include <linux/types.h>
 #include <linux/if_arp.h>
 #include <linux/if_ether.h>
